src/app/main/dialog/dialog.component.html

```diff
@@ -212,11 +204,15 @@
         <app-dialog-header [title]="data.title" [subtitle]="data.subtitle" (close)="dialogRef.close()">
```

Comment:
This header still has the `(close)` parameter

Reply:
fixed in ee60add